spyder/plugins/editor/api/folding.py

```diff
@@ -21,12 +21,16 @@
 
 # Standard library imports
 import sys
+import logging
 
 # Local imports
 from spyder.plugins.editor.utils.editor import TextBlockHelper
 
+logger = logging.getLogger(__name__)
 
-def print_tree(editor, file=sys.stdout, print_blocks=False, return_list=False):
+
+def print_tree_triggers(editor, file=sys.stdout, print_blocks=False,
```

Comment:
This function seems almost the same as `print_tree` below and it's not used anywhere now. Please remove it if it's not necessary anymore.